eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FieldValue/Handler.php

```diff
@@ -147,4 +157,50 @@ protected function lowerCase(string $string): string
     {
         return $this->transformationProcessor->transformByGroup($string, 'lowercase');
     }
+
+    /**
+     * @param mixed $value
+     *
+     * @return mixed
+     */
+    protected function prepareParameter($value)
+    {
+        if (is_string($value)) {
+            return $this->lowerCase($value);
+        } elseif (is_array($value)) {
+            return array_map([$this, 'prepareParameter'], $value);
+        }
+
+        return $value;
+    }
+
+    protected function createNamedParameter(QueryBuilder $outerQuery, string $column, $value): ?string
```

Comment:
```suggestion
    private function createNamedParameter(QueryBuilder $outerQuery, string $column, $value): ?string
```

Reply:
I actually had this exact comment to the original PR :sweat_smile: